apps/cli/src/commands/rollups/start.ts

```diff
@@ -231,15 +231,20 @@ export const createStartCommand = () => {
                           services.includes(name),
                       );
 
-            // add to compose files list
-            composeFiles.push(...optionalServices.map(({ file }) => file));
+            // validate services and add to compose files
+            for (const service of optionalServices) {
+                if (!availableServices.includes(service)) {
```

Comment:
the list is already filtered by "includes", so this section is not really doing anything